src/oas3/transformers/content.ts

```diff
@@ -20,7 +20,7 @@ function translateEncodingPropertyObject(
   encodingPropertyObject: EncodingPropertyObject,
   property: string,
 ): IHttpEncoding {
-  const acceptableStyles: Array<string | undefined> = [
+  const acceptableStyles: (string | undefined)[] = [
```

Comment:
Ha, cool

Reply:
This was refactored automagically by tslint, I didn't touch that